platforms/minestom/src/main/java/com/dfsek/terra/minestom/MinestomPlatform.java

```diff
@@ -45,18 +49,32 @@ public void register(TypeRegistry registry) {
     @Override
     public boolean reload() {
         getTerraConfig().load(this);
-        boolean succeed = loadConfigPacks();
+        getRawConfigRegistry().clear();
+
+        try {
+            getRawConfigRegistry().loadAll(this);
+        } catch (IOException e) {
+            LOGGER.error("Failed to load configurations due to I/O error", e);
+            return false; // reload failed
+        } catch (PackLoadFailuresException e) {
+            LOGGER.error("Failed to load configurations due to pack load failures", e);
+            return false; // reload failed
+        } catch (Exception e) {
+            // Catch any other exceptions that might be thrown
+            LOGGER.error("Failed to load configurations due to unexpected error", e);
+            return false;
+        }
```

Comment:
Why was this pulled out of the common function? If changes need to be made please make improve the existing code (`loadConfigPacks`) rather than duplicating logic